### PR TITLE
valgrind fixes

### PR DESCRIPTION
**Pull request by badhrinathpa**

### CODE REVIEW

**Comment by badhrinathpa** [comment]
retest this please

**Comment by badhrinathpa** [comment]
retest this please

**Comment by badhrinathpa** [comment]
retest this please

File: src/s1ap/handlers/s1ap_msg_delegator.c [hunk 1]
```diff
@@ -278,15 +278,21 @@ handle_s1ap_message(void *msg)
     switch (pdu_p->present) {
         case S1AP_PDU_PR_initiatingMessage:
             s1ap_mme_decode_initiating (pdu_p->choice.initiatingMessage, enb_fd);
-            free(pdu_p->choice.initiatingMessage);
+            //free(pdu_p->choice.initiatingMessage);
```
Comment (thakurajayL) [inline src/s1ap/handlers/s1ap_msg_delegator.c:281]: Do you want to remove these comments ?

File: src/s1ap/handlers/s1ap_encoder.c [hunk 2]
```diff
@@ -113,31 +133,37 @@ int s1ap_mme_encode_service_rej(
 
     log_msg(LOG_DEBUG, "Encode Serivce Rej");
     pdu.present = S1AP_PDU_PR_initiatingMessage;
-    pdu.choice.initiatingMessage = calloc(sizeof(InitiatingMessage_t), sizeof(uint8_t));
+    pdu.choice.initiatingMessage = //calloc(sizeof(InitiatingMessage_t), sizeof(uint8_t));
```
Comment (thakurajayL) [inline src/s1ap/handlers/s1ap_encoder.c:136]: Does this compile ? I see code comment after =

Comment (badhrinathpa) [reply src/s1ap/handlers/s1ap_encoder.c]: fixed. not sure how it was compiling.

File: src/s1ap/handlers/s1ap_msg_delegator.c (hunk 1 above)
Comment (badhrinathpa) [reply src/s1ap/handlers/s1ap_msg_delegator.c]: fixed

**Review by thakurajayL — APPROVED** [review]

**Pull request merged**